app/controllers/permits_controller.rb

```diff
@@ -0,0 +1,74 @@
+class PermitsController < ApplicationController
+  before_action :set_permit, only: [:show, :edit, :update, :destroy]
+
+  # GET /permits
+  # GET /permits.json
+  def index
+    @permits = Permit.all
+  end
+
+  # GET /permits/1
+  # GET /permits/1.json
+  def show
+  end
+
+  # GET /permits/new
+  def new
```

Comment:
Permits will not be created, edited, or destroyed using this viewer, so the routes are unnecessary.